tests/nuodb_cursor_test.py

```diff
@@ -125,6 +127,29 @@ def test_executemany_somefail(self):
 
         cursor.execute("DROP TABLE executemany_table")
 
+    def test_result_set_gets_closed(self):
+        current_version = getenv('NUODB_VERSION', None) # skipif <2.1
+        if current_version is not None and not LooseVersion(current_version) < LooseVersion("2.1"):
+        # Server will throw error after 1000 open result sets
```

Comment:
This comment refers to if the above if statement is not true, correct? Not sure that's clear.